### ISSUE TITLE
Render query variables in the sql custom checks

### ISSUE THREAD

**Issue by hishoss** [redata-team/redata#69]
**Tell us about the problem you're trying to solve**
In the custom query check functionality, query strings containing variables eg. `select col_a from {{table_name}}` are currently failing when these variables are written a bit differently than expected eg. `{{ table_name }}`. This is due to [check_custom_query](https://github.com/redata-team/redata/blob/a43ca500dc82a3d3ad5c285869601d956edc4d0f/redata/backends/base.py#L48) function expecting the name of these variables in exactly the same format within the code, which then uses a string replace.

**Describe the solution you’d like**
This is easily solved by a templating framework like jinja2. For eg: you can easily write 
1. `select col_a from {{table_name}}`
2. `select col_a from {{ table_name }}`, etc.

which are all valid. We'll have to update the check_custom_query with something along the lines

```
from jinja2 import Template

def check_custom_query(self, table, query, conf):
       ...
        query = Template(query)
        query = query.render(
            table_name=table.full_table_name,
            period_start=period_start,
            period_end=period_end
        )
```

**Describe the alternative you’ve considered or used**
An alternative can be to use a regex pattern.

**Comment by mateuszklimek** [comment]
@hishoss Totally agree doing it with jinja2 sounds like best option.

**Comment by mateuszklimek** [comment]
Solved by #86
